nextflow.config

```diff
@@ -35,41 +35,66 @@ params {
 
 profiles {
 
-  standard {
-        params.cloudProcess = false
-        params.cloudDatabase = false
-        includeConfig 'configs/local.config'
-  }
-
- lsf {
-        params.cloudProcess = true
-        includeConfig 'configs/lsf.config'
- }
-
- ebi {
-        params.cloudProcess = true
-  	  params.workdir = "/hps/nobackup2/production/metagenomics/$USER/nextflow-work-$USER"
-  	  params.cloudDatabase = "/homes/$USER/data/nextflow-databases/"
-  	  params.cachedir = "/hps/nobackup2/singularity/$USER"	
-        includeConfig 'configs/lsf.config'
- }
-
- git_action {
-        params.cloudProcess = false
-        params.cloudDatabase = false
-        includeConfig 'configs/local.config'
-        params.dv = true
-        params.ma = true
-        params.mp = true
-        params.vf = false
-        params.vs = true
-        params.pp = true
-        params.sm = true
-        params.vb = true
-        params.vn = true
-        params.anno = true
-        params.cores = 2
-        params.fasta = 'test-data/all_pos_phage.fa'
- }
+       local {
+              workDir = params.workdir
+              params.cloudProcess = false
+              includeConfig 'configs/local.config'
+       }
+
+       lsf {
+              executor {
+                     name = "lsf"
+                     queueSize = 200
+              }
+              workDir = params.workdir
+              params.cloudProcess = true
+              process.cache = "lenient"
+              includeConfig 'configs/node.config'
+       }
+
+       docker {
+              docker { enabled = true }
+              includeConfig 'configs/container.config'
+       }
+
+       singularity {
+              singularity { 
+                     enabled = true
+                     autoMounts = true
+                     cacheDir = params.cachedir
+              }
+              includeConfig 'configs/container.config'
+       }
+
+       git_action {
+              params.cloudProcess = false
+              includeConfig 'configs/local.config'
+              params {
+                     ma = true
+                     mp = true
+                     vf = false
+                     vs = true
+                     pp = true
+                     sm = true
+                     vb = true
+                     vn = true
+                     dv = true
+                     anno = true
+                     fasta = 'test-data/all_pos_phage.fa'
+              }
+
+       }
+
+// test configs
+
+
+
+       ebi {
```

Comment:
If you want, you can directly change this to 

```java
       ebi {
              executor {
                     name = "lsf"
                     queueSize = 200
              }
              process.cache = "lenient"
              params.cloudProcess = true
              params.workdir = "/hps/nobackup2/production/metagenomics/$USER/nextflow-work-$USER"
              params.databases = "/homes/$USER/data/nextflow-databases/"
              params.cachedir = "/hps/nobackup2/singularity/$USER"	
              includeConfig 'configs/node.config'
       }
```

Reply:
adding this